workflow/common/security_context.go

```diff
@@ -0,0 +1,26 @@
+package common
+
+import (
+	corev1 "k8s.io/api/core/v1"
+	"k8s.io/utils/pointer"
+)
+
+func ContainerSecurityContext() *corev1.SecurityContext {
```

Comment:
```suggestion
func MinimalCtrSC() *corev1.SecurityContext {
```

I think we should be more specific here with "minimal".

The other part is just shortening the name with common abbreviations